### DISCUSSION TITLE
Mac Ventura - user permissions on /usr/local/ and /opt are changing on reboot

### DISCUSSION THREAD

**Discussion by ArchaicLord** [Homebrew/discussions#3896] [Everyday usage]
Hello,

I don't think this is a Brew-specific issue so I was hoping someone might be able to point me in a better direction to help.
I wrote a script for our users that sets the required file permissions to allow them to install brew.
(FYI - our users do not get full admin privilege,  so this is why we had to take this approach)

Since the new MacOS update, it seems permissions on /usr/local for Intel and /opt for the M1s are being reset back to root on reboot 100% of the time. One user reported the permission change before rebooting. (Not a behaviour I have seen myself)

I tried using fs_usage utility but this only works realtime. I tried setting it to run and log to file in the background but that was fruitless.

It is not a login issue. I set the shell to both Bash and ZSH. The outcome was the same.

Would anyone be able to help me track down the cause? Googling just tells me how to set the permission.
Is there anything else I can use? 

What would you suggest would be a good way to counteract this? 
Many thanks

**Comment by p-linnane** [comment]
This is not an issue with Homebrew. Without knowing what's in your script, I can tell you that there is [a bug](https://www.malwarebytes.com/blog/news/2022/11/macos-ventura-bug-disables-security-software) in macOS 13.0 relating to disk access. It may be related, it may not, but we will not be able to provide more guidance since it's not something we can fix with `brew`.
